src/components/ec-modal/ec-modal.vue

```diff
@@ -222,151 +225,135 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/settings/colors/index';
-@import '../../scss/tools/index';
-
-$ec-modal-bg: rgba($level-2-doc-bodies-bg, 0.2) !default;
-$ec-modal-color: $level-3-body-and-headings !default;
-$ec-modal-content-bg: $white !default;
-$ec-modal-content-footer-bg: $level-7-backgrounds !default;
-$ec-modal-close-btn-fill: $level-4-interactive-elements !default;
-$ec-modal-close-btn-fill-hover: $level-4-tech-blue !default;
+<style>
+@import '../../styles/tools/scrollbars.css';
+@import '../../styles/tools/transitions.css';
 
 .ec-modal {
-  background: $ec-modal-bg;
-  width: 100vw;
-  height: 100vh;
-  position: fixed;
-  top: 0;
-  left: 0;
-  color: $ec-modal-color;
+  background: hsla(var(--ec-gray-color-level-2), 0.2);
 
-  @include z-index-modal;
+  @apply tw-w-screen;
+  @apply tw-h-screen;
+  @apply tw-fixed;
+  @apply tw-inset-0;
+  @apply tw-text-gray-3;
+  @apply tw-z-modal;
 
   &__content {
-    width: calc(100% - 24px);
-    max-width: 680px;
-    position: relative;
-    top: 50%;
-    left: 50%;
-    background: $ec-modal-content-bg;
+    width: calc(100% - theme('spacing.24'));
+    max-width: var(--ec-modal-sm-max-width);
+    max-height: calc(100% - theme('spacing.24'));
     transform: translate(-50%, -50%);
-    max-height: calc(100% - 24px);
-    overflow-y: auto;
 
-    @include box-shadow-level-2;
-    @include scrollbar;
-    @include shape-border-radius;
+    @apply tw-relative;
+    @apply tw-inset-1/2;
+    @apply tw-bg-gray-8;
+    @apply tw-overflow-y-auto;
+    @apply tw-shadow-level-2;
+    @apply tw-rounded;
+
+    @mixin md-scrollbar;
 
-    @media screen and (min-height: 568px) {
-      display: flex;
-      flex-direction: column;
+    @media screen and (min-height: theme('screens.sm')) {
+      @apply tw-flex tw-flex-col;
     }
 
-    @include media__from-1024 {
+    @screen lg {
       width: 60vw;
-      max-width: 700px;
```

Comment:
Curious to know why this was removed.

Reply:
@mcibique trying to use as less abstract numbers as possible. We already have a  --ec-modal-sm-max-width: 680px; shall I add another one for 700px? :/

Reply:
Spoke with Uve 680 is ok by her so would consider this comment resolved :)